samples/highcharts/demo/donut/demo.js

```diff
@@ -0,0 +1,39 @@
+Highcharts.chart('container', {
+    chart: {
+        type: 'pie',
+        backgroundColor: '#222'
```

Comment:
@oysteinmoseng Regarding explicit dark mode demos - should we continue having that now that we have the adaptive theme and a light/dark switch in the UI? The pro is that we get to display theming at a glance. The con is that this demo is locked in dark mode even if users opt for light mode.

Reply:
If we choose to go for explicit dark mode, the background color should be `#141414` to match the adaptive theme.

Reply:
I think it's important that not all of these basic demos use the same theme - we need to show a wide array of styling. We should however also show off best practice for theming. With all the recent changes I am not actually sure what the best practice is (presumably something that works with light & dark mode), so maybe you can chime in on that part...